From e469e34954dc8eb27b0f2fe521e18ac2e03dad61 Mon Sep 17 00:00:00 2001
From: "M. Vefa Bicakci" <vefa.bicakci@windriver.com>
Date: Thu, 3 Feb 2022 21:34:14 -0500
Subject: [PATCH] Enable CONFIG_PAGE_POOL by default

CONFIG_PAGE_POOL is a "hidden" configuration option, which causes it to
be enabled only when another kernel configuration option explicitly
selects it. Specifying CONFIG_PAGE_POOL=y in .config does not have any
effect.

Given that StarlingX disables all in-tree device drivers that explicitly
select CONFIG_PAGE_POOL, and given that at least one out-of-tree kernel
module (mlnx-ofa_kernel) requires the features made available by
CONFIG_PAGE_POOL, this commit enables the configuration option in
question by default.

Signed-off-by: M. Vefa Bicakci <vefa.bicakci@windriver.com>
---
 net/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/Kconfig b/net/Kconfig
index 05b0f041f039..62443980ba6b 100644
--- a/net/Kconfig
+++ b/net/Kconfig
@@ -437,6 +437,7 @@ config NET_DEVLINK
 
 config PAGE_POOL
 	bool
+	default y
 
 config FAILOVER
 	tristate "Generic failover module"
-- 
2.29.2

